.travis.yml

```diff
@@ -2,4 +2,5 @@ language: go
 go:
   - 1.6
   - 1.7
+  - 1.12
```

Comment:
supporting last 2 versions and no tip so update accordingly.

Reply:
done

Reply:
use latest of each